### ISSUE TITLE
Check the code at W3C - WCAG

### ISSUE THREAD

**Issue by mathieu-bonhommeau** [matanna/MathieuBonhommeau_6_07122021#11]
- Check the code at W3C
- Check the code with AChecker. No "know issue" should be detected.
- Check the code with other checker (Wave, ...)

**Comment by mathieu-bonhommeau** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Vérifier le code et l'accessabilité](https://trello.com/c/apJffrOJ/15-v%C3%A9rifier-le-code-et-laccessabilit%C3%A9)
